configman/def_sources/for_argparse.py

```diff
@@ -573,7 +575,7 @@ def parse_args(self, args=None, namespace=None):
             values_source_list=self.value_source_list,
             argv_source=args,
             app_name=self.prog,
-            app_version=self.version,
+#            app_version=self.version,
```

Comment:
Mmm... I think we should be passing it in regardless of Python version. Version is helpful to know when using a cli.

I'm not sure why there is no `version` attribute. There should be. I looked at the Python 3.2 source and `ArgumentParser.__init__` sets it on the instance based on incoming arguments. How puzzling. I'll take a look at this more after I finish reviewing.